main.go

```diff
@@ -62,11 +66,19 @@ func main() {
 		setupLog.Error(err, "unable to start manager")
 		os.Exit(1)
 	}
-
+	// TODO: Set this with credentials on the server side
+	// TODO: This needs to be set so we don't fail when instantiating client
+	conn, err := grpc.DialContext(context.Background(), "porter-grpc-service:3001", grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
What I can do to not be dependent on the gRPC server being there is to only set this if the service is present.  

Basically, don't error if this fails but pass an empty listener and log "no grpc client for installationoutputs" @schristoff

Reply:
Sgtm